data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard1.json

```diff
@@ -106,5 +106,28 @@
     "id": "TALK_EVAC_GUARD1_BYE",
     "dynamic_line": "Stay safe out there.  Hate to have to kill you after you've already died.",
     "responses": [ { "text": "...", "topic": "TALK_DONE" } ]
+  },
+  {
+    "id": "MISSION_EVAC_SMOKER_GET_CIGARETTES",
+    "type": "mission_definition",
+    "name": "Smokes, let's go.",
+    "description": "Make a friend at the refugee center by handing over around four packs - 100 cigarettes.",
```

Comment:
That would be around five packs, since cigarettes almost always come twenty to a package.

Reply:
Will adapt that at some point. I think I even checked wikipedia for US amounts and just put it down wrong :)